portfolio/src/main/java/com/google/sps/servlets/Login.java

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 Google LLC
```

Comment:
This file starts to mix concerns on the server. So far our servlets have all served up JSON and been kind of an API while this servlet provides dual functionality. `doGet` now serves HTML and `doPost` also does in a round-about way. `doHead` just serves a status, but really acts more like an API than something that serves HTML.

In a real app you'd want to separate servlets that produced HTML/CSS/JavaScript from those that were used as an API.

I feel like we are going to need to spend some time to think through this file and split it into a few files. I've added some comments on how to get started. We'll need to refactor after this PR goes in though.